src/frontends/pytorch/include/openvino/frontend/pytorch/node_context.hpp

```diff
@@ -54,6 +54,8 @@ class NodeContext : public frontend::NodeContext {
         return m_tensor_map->at(input);
     }
 
+    Any const_input(int index) const override;
```

Comment:
This version returns ov::Any so it can be used by visitor in common frontend extension code:
https://github.com/openvinotoolkit/openvino/pull/16434/files#diff-a441f7a33d0e530dc1b19988cda0dd386a5cde256bf34871dc30217b9de8ba10R243

It's analogous to NodeContext::get_attribute_as_any.

Reply:
cant we use `const_input<Any>(index)` instead of creating a new method? Or maybe we can have a different name for it? e.g. `const_input_as_any`?

Reply:
const_input_as_any looks good to me. I renamed that function.